doc/administrator/mapfile.rst

```diff
@@ -12,12 +12,12 @@ for the map (``WMS GetMap``), and for the ``point query`` feature (``WMS GetFeat
 WFS is used for the ``query`` (box and point), the ``filters``, and the WFS permalink features.
 
 The application's mapfile is located in the ``mapserver`` directory, it is
-commonly named ``mapserver.map.mako``.
+commonly named ``mapserver.map.tmpl``.
 
 .. note::
 
-    The application's mapfile is a ``.mako`` file because it contains variables.
-    These variables are substituted when the ``make`` build command is executed.
+    The application's mapfile is a ``.tmpl`` file because it contains variables.
+    These variables are substituted the container starts.
```

Comment:
These variables are substituted when the container starts.

or

These variables are substituted at container start.